### PR TITLE
Add a gradle task to automate dash-licence checks

### PR DESCRIPTION
**Pull request by borisrizov-zf**
## Description

Add gradle tasks to download the dash-licence tool, get a flat map of the project dependencies and run the dash-tool on the input.
This PR also adds a plugin to grade which allows loading env variables from an .env file - good for local development and quick tests.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ x ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ x ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: src/test/java/org/eclipse/tractusx/managedidentitywallets/config/TestContextInitializer.java [hunk 1]
```diff
@@ -50,6 +50,7 @@ public void initialize(ConfigurableApplicationContext applicationContext) {
                 "server.port=" + port,
                 "miw.host: localhost:${server.port}",
                 "miw.enforceHttps=false",
+                "miw.vcExpiryDate=1-1-2030",
```
Comment (carslen) [inline src/test/java/org/eclipse/tractusx/managedidentitywallets/config/TestContextInitializer.java:53]: Is this line related to the topic of this PR?

File: .github/workflows/app-test-coverage.yml
[the remark this answers is not in the document]
Comment (borisrizov-zf) [reply .github/workflows/app-test-coverage.yml:1]: I think this came in through the rebase onto the develop branch, I'll double-check tho.

File: src/test/java/org/eclipse/tractusx/managedidentitywallets/config/TestContextInitializer.java (hunk 1 above)
Comment (borisrizov-zf) [reply src/test/java/org/eclipse/tractusx/managedidentitywallets/config/TestContextInitializer.java:53]: Same as the workflow, must be a rebasing issue. I'll double check.

**Comment by borisrizov-zf** [comment]
@carslen The issues you mentioned were related to the fact that I was using "main" as base. Now there are no funny extras.

**Comment by carslen** [comment]
LGTM now but PR is marked as Draft. Ready for review/merge?

**Comment by borisrizov-zf** [comment]
I will remove the draft status as soon as #87 is merged and I have rebased my branch.

**Review by DominikPinsel — APPROVED** [review]
